large_page-c/example/Makefile

```diff
@@ -21,7 +28,7 @@ OBJFILES=              \
   filler16.o           \
 
 OBJS=$(addprefix $(OBJDIR)/,$(OBJFILES))
-LDFLAGS=-Wl,-T ../ld.implicit.script -Wl,-z,max-page-size=2097152
+LDFLAGS=-Wl,-z,max-page-size=2097152
```

Comment:
OOI, if the linker script is not portable, are the (what look like) gnu ld-ish specific `-Wl` options fairly portable?

Reply:
That's a good point. So far we've only been concerned with linkers from gcc, clang, and gold. We should test to make sure the example works with all of these.

Reply:
I agree we should take out the -z option. This is instructing the linker to use 2M. I am not sure why we added this.